README.md

```diff
@@ -80,6 +80,87 @@ Use the command:
 If the test fails, look in the log files in the same directory.
 Here  <install_dir> is the Intel MLSL installation directory.
 
+## Migration to oneCCL ##
+
+Intel® MLSL is no longer supported, no new releases are available, please switch to the new API introduced in Intel® oneAPI Collective Communications Library (oneCCL) http://github.com/intel/oneccl.
```

Comment:
Intel® MLSL is no longer supported, no new releases are available. Please switch to the new API introduced in [Intel® oneAPI Collective Communications Library (oneCCL)](http://github.com/intel/oneccl).

Reply:
Also, please add a note in the beginning of a readme file with the same line of text. Before the introduction section.